### ISSUE TITLE
Add conda-forge feedstock

### ISSUE THREAD

**Issue by chrisjsewell** [jupyter/jupyter-sphinx#148]
I can probably do this if you want?

**Comment by akhmerov** [comment]
Good idea. If you're up for it, that'd be awesome, otherwise I'll do it once I figure everything out (didn't set up conda forge yet).

**Comment by chrisjsewell** [comment]
see https://github.com/conda-forge/staged-recipes/pull/12383

**Comment by akhmerov** [comment]
Thanks a lot! I'll take care of the docs and close the issue.

**Comment by akhmerov** [comment]
eh there's now https://github.com/conda-forge/jupyter_sphinx-feedstock and https://github.com/conda-forge/jupyter-sphinx-feedstock ...

And the README also referred to the original one :rofl:

**Comment by chrisjsewell** [comment]
Ah doh I missed that because was searching for jupyter-sphinx not jupyter_sphinx 🤦 
It is normally common practice though IMO, to use the `-` version.
I guess if you pick one, and we can ask conda-forge to remove the other?

Also note your README is wrong, because it says `pip install jupyter_sphinx`, rather than `pip install jupyter-sphinx` (its correct on RTD)

and another nitpick, you should and the RTD site to the GitHub "About" box

**Comment by akhmerov** [comment]
> and another nitpick, you should and the RTD site to the GitHub "About" box

I'll have to ask maintainers for that again (@choldgraf @SylvainCorlay).

About the feedstock my plan now is to provide both `jupyter_sphinx` and `jupyter-sphinx` via the `jupyter-sphinx` feedstock, and then archive the `jupyter_sphinx` one.

**Comment by choldgraf** [comment]
@akhmerov I just switched the jupyter-sphinx group to have maintainer status rather than just write status...does that allow you to add tags etc?

**Comment by akhmerov** [comment]
Yep, thanks a lot! Fixed now.

**Comment by 12rambau** [comment]
Can someone with admin right can add me as maintainer on these 2 repositories ? I started making updates in the pypi distribution but the conda one is still stuck a v0.4.0

**Comment by akhmerov** [comment]
Done, thanks again for your effort!
